Dockerfile

```diff
@@ -42,6 +42,8 @@ ENV PORT= \
     TZ=UTC \
     SERVER_DESCRIPTION= \
     BACKUP_ENABLED=true \
+    DELETE_OLD_BACKUPS=false \
+    OLD_BACKUP_DAYS=30 \
```

Comment:
Moved the default value here as it was it was the only one whose default value was in a script